### DISCUSSION TITLE
제페토  UI

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#954] [5. Q&A]
Zepeto Studio에서 유니티로 월드를 만들고 있습니다.
제공된 example들을 보면 ui는 단순해보이는데, zepeto에서 unity application을 독립적으로 빌드하는 건가요?
기존의 zepeto ui를 사용할 수는 없는 건가요? 

https://github.com/naverz/zepeto-studio-kor/discussions/895
여기 디스커션에 보면 기본적인 ui는 제페토 스튜디오에서는 제공되지 않는 것으로 보입니다.

**Comment by Sonak0930** [comment]
기존 Zepeto 앱에 올라오는 월드는 유니티가 아니면 어떻게 만든 건가요>?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 제페토 공식 월드 및 빌드잇에 사용된 UI는 정해진 포맷에 맞춰서 생성됩니다.
하지만 유니티 월드는 크리에이터분들이 UI 등으로 다양하게 활용하실 수 있도록 홈, 채팅 UI만 기본적으로 생성됩니다. (ex.슬라임파티)
카메라 기능은 직접 저희가 제공한 API를 사용해서 타입스크립트로 구현하실 수 있습니다.
가이드를 참고하여 주세요.
https://studio.zepeto.me/kr/guides/screenshot

그 외 니즈가 많은 제스쳐, 음성 채팅 등의 기능은 점차적으로 지원할 예정입니다.

공식 맵의 구현 방법은 내부 사항이라 말씀 드리기 어려운 점 양해 부탁 드립니다.
감사합니다.
